README.md

```diff
@@ -93,196 +66,166 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
 
-### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+### Key Features <a name="key-features"></a>
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Add_bike**
+- **Delete_bike**
+- **Fetch/get_bike**
+- **Authenticate_user**
+- **Reserve_bike**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+## API-Documentation <a name="api-documentation"></a>
+
+- [Link To Documentation](https://bike-rental-api.onrender.com/api-docs/index.html)
+
+
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- [Live Demo Link](rentbikes.netlify.app)
 
-- [Live Demo Link](https://google.com)
+## Back-end Repository <a name="backend-repo"></a>
```

Comment:
- [ ] The backend repo link leads to a 404 page.  Kindly either correct it or remove it in order to prevent user confusion. 👍